web_src/js/components/VueComponentLoader.js

```diff
@@ -8,13 +8,14 @@ export function initVueEnv() {
   if (vueEnvInited) return;
   vueEnvInited = true;
 
-  const isProd = window.config.runModeIsProd;
-  Vue.config.productionTip = false;
-  Vue.config.devtools = !isProd;
+  // As far as I could tell, this is no longer possible.
+  // But there seem not to be a guide what to do instead.
+  // const isProd = window.config.runModeIsProd;
+  // Vue.config.devtools = !isProd;
```

Comment:
I think this option was moved to compile-time config, https://github.com/vuejs/core/tree/main/packages/vue#bundler-build-feature-flags, but it might just be that it's no longer necessary to get rid of devtools in prod build as the docs sound like it's disabled by default.

Reply:
Hm, compile-time config won't be evaluated at runtime, though. That is, I can't go via `window.config` anymore.

The value is injected in

https://github.com/go-gitea/gitea/blob/0e8c81e16121af19dfb66aae882284a50b1b24af/templates/base/head_script.tmpl#L14

which in turn is defined in

https://github.com/go-gitea/gitea/blob/0e8c81e16121af19dfb66aae882284a50b1b24af/modules/context/context.go#L703

which in turn is derived from

https://github.com/go-gitea/gitea/blob/0e8c81e16121af19dfb66aae882284a50b1b24af/modules/setting/setting.go#L1015-L1016

… so … is there a way I can somehow read this from Node.js?